client/src/components/SideInfoPanal/SideInfoPanal.jsx

```diff
@@ -8,7 +8,6 @@ import GamesContext from '../../context';
 
 const SideInfoPanal = () => {
   const { sidePanalImg, sidePanalInfo } = useContext(GamesContext);
-  console.log(sidePanalInfo);
 
```

Comment:
On line 15, you are using an object.subObject. The point of destructuring is to make it so you don't have to type longer names. May be worth considering if you can bring it in as just short_description somehow. That's very minor though. Feel free to ignore.

Reply:
That's a good point. I was actually doing it that way but the side panel uses so many properties it was cleaner to just restructure this and use the properties each explicitly.